index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+   <div class="page-wrapper">
+     <section class="head-top">
+       <div class="header-log">
+        <h2>My Logo</h2>
+       </div> 
+       <nav class="main-nav">
+        <a href="#">☰</a>
+        <ul class="nav-wrapper">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+       </nav>
+     </section> 
```

Comment:
Nice job adding a `nav` element! Although, please make sure to also use a `header` (instead of this `section`). Kindly notice that the [header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) element should contain your logo (it can be just a text for now) and a [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element with your navigation links (in this case, your hamburger icon) 😉. By knowing this, please update your `header` to make your project semantically accurate and improve your page accessibility.   

Also, notice that your Logo shouldn't be a `h2`. Instead, use an `a` tag.

You can have something like this:  

```html 
<header> 
  <a href=#>My Logo</a>  

  <nav> 
    Hamburger icon 
  </nav> 
</header> 
```

Reply:
Edited Header section and replaced h2 element with a elament